rdkcentral/rdkservices
# RDK-35340: Update System Service  Definition

Reason for change: updated the getPublicIP api to allow only interface and ipv6 parameter and also update document changes
Test Procedure: please referred from RDK-35348
Risks: Low
Signed-off-by: Thamim Razith <ThamimRazith_AbbasAli@comcast.com>
(cherry picked from commit 532965865700be41c7574cf5d937841f054c665d)

RDK-35340: Update System JSON Definition

Reason for change: Update the JSON Schema to include updated methods and generate md accordingly.
Test Procedure: verify it SystemPlugin.md
Risks: Medium
Signed-off-by: Kumar Santhanam <Kumar_Santhanam@comcast.com>
(cherry picked from commit 2d6613f445021200f605125e51a7b67d15589058)
